.gitmodules

```diff
@@ -293,4 +288,9 @@
 	path = third_party/open-iot-sdk/storage
 	url = https://git.gitlab.arm.com/iot/open-iot-sdk/storage.git
 	branch = main
-    platforms = openiotsdk
+	platforms = openiotsdk
+[submodule "bouffalolab_sdk"]
+	path = third_party/bouffalolab/repo
+	url = https://github.com/bouffalolab/bl_iot_sdk_tiny.git
```

Comment:
naming is your choice as a platform, however from an external review feedback: `tiny` does not tell me what it actually is ... maybe `no-toolchain` or similar would be more obvious.

Reply:
repo bl_iot_sdk_tiny is dedicated to reduce code size and SRAM size at our SDK side. It is a different SDK, not jsut removing toolchain.